tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2360,10 +2388,12 @@ snmp/test_snmp_queue.py:
 snmp/test_snmp_queue_counters.py:
   skip:
     conditions_logical_operator: OR
-    reason: "Have an known issue on kvm testbed / Unsupported in M* topos"
+    reason: "Have an known issue on kvm testbed / Unsupported in M* topos or issue #17929 on Mellanox platform"
     conditions:
       - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/14007"
       - "topo_type in ['m0', 'mx', 'm1']"
+      - "https://github.com/sonic-net/sonic-mgmt/issues/17929 and asic_type in ['mellanox']"
```

Comment:
not seeing this failure in MS.

Reply:
@AharonMalkin , is this one still failing on NVidia side? I think we can remove this one now. the issue should be already fixed.

<img width="649" height="45" alt="image" src="https://github.com/user-attachments/assets/14596266-3520-4342-8598-0f49f1511692" />

Reply:
Still fails on 202412